app/views/helpers/index/normal/entry_share_menu.phtml

```diff
@@ -1,29 +1,26 @@
 <?php
 	declare(strict_types=1);
 	/** @var FreshRSS_View $this */
-	//TODO: To reduce output size, replace by an on-demand JavaScript menu, potentially using HTML template element
 ?>
 <ul class="dropdown-menu">
 	<li class="dropdown-header"><?= _t('index.share') ?> <a href="<?= _url('configure', 'integration') ?>"><?= _i('configure') ?></a></li><?php
-		$id = $this->entry->id();
-		$link = $this->entry->link();
-		$title = $this->entry->title() . ' · ' . ($this->feed === null ? '' : $this->feed->name());
+		$link = '--link--';
+		$title = '--titleText----websiteName----articleAuthors--';
 		foreach (FreshRSS_Context::userConf()->sharing as $share_options) {
 			$share = FreshRSS_Share::get($share_options['type']);
 			if ($share === null) {
 				continue;
 			}
 			$cssClass = $share->isDeprecated() ? ' error' : '';
-			$share_options['id'] = $id;
```

Comment:
Regression https://github.com/FreshRSS/Extensions/issues/263#issuecomment-2546636861

Reply:
Great. That helped to fix it

Reply:
https://github.com/FreshRSS/FreshRSS/pull/7113